LiteCore/Database/LiveQuerier.cc

```diff
@@ -41,24 +51,83 @@ namespace litecore {
     ,_continuous(continuous)
     ,_delegate(delegate)
     {
-        logInfo("Created on Query %p", query);
+        logInfo("Created on Query %s", query->loggingName().c_str());
```

Comment:
is this `loggingName` or `loggingIdentifier`?

Reply:
`loggingName` is correct. It returns the `{Query#123}` string that shows up in logs.